### ISSUE TITLE
timezone can be confusing, leading to "voting is not yet open"

### ISSUE THREAD

**Issue by AcBdSc** [benadida/helios-server#393]
I am getting a 'voting is not yet open' message, and ballots cannot be cast, despite the poll having been frozen and the start date for voting has passed. I see no other setting affecting this.

**Comment by snoop168** [comment]
did you add voters?

**Comment by AcBdSc** [comment]
In my case it was a time zone issue, ie my time zone is an hour behind Helios (you may have found the same thing by now).

**Comment by benadida** [comment]
yeah, the timezone stuff is not as clear as it should be. I'm going to update this issue to reflect that.

**Comment by laoumh** [comment]
@benadida Django now has all the tools to show some datetime for each user at their local time — https://docs.djangoproject.com/en/5.1/topics/i18n/timezones/#time-zone-aware-output-in-templates

For example:

- An election administrator in New York sets the voting start to June 23, 2025, 8 a.m. (implicitly -04:00 UTC)
- The Helios server saves as June 23, 2025, noon (UTC time)
- A voter in Tokyo will see the start time as June 23, 2025, 9 p.m. (+9:00 UTC)

Of course, that's a change in behavior for people already familiar with Helios using UTC as the time reference.

I did a related maintenance on my University version of the system to have "São Paulo" as the system reference. The hardest part was getting `SplitSelectDateTimeWidget`to behave correctly, since its a custom widget and so Django doesn't perform the  conversion automatically, it has to be done explicitly.
